1-exercises/A-reading-list/script.js

```diff
@@ -1,27 +1,52 @@
 function readingList(books) {
-  // Write your code here...
+  // Write your code here..
+  const getParentElement = document.querySelector("#content");
+  
+  const ulElement = document.createElement("ul");
+
+  for (let book of books) {
+    const liElement = document.createElement("li");
```

Comment:
Would make more descriptive names if possible e.g. bookListItem so you can tell what the varible is for